tests/trade_match/test_trade_match.py

```diff
@@ -0,0 +1,174 @@
+import pytest
+from playwright.sync_api import expect, Page
+from vega_sim.service import VegaService
+
+from actions.vega import submit_multiple_orders
+
+
+@pytest.mark.usefixtures("opening_auction_market", "auth")
+def test_trade_match_table(opening_auction_market: str, vega: VegaService, page: Page):
+    positions_button = page.get_by_test_id("Positions")
```

Comment:
Where these aren't getting reused a lot / if at all. Is it exccessive to pull these selectors out?